src/legacy/core_plugins/kibana/public/dashboard/dashboard_app.tsx

```diff
@@ -63,7 +63,8 @@ import { Filter } from '@kbn/es-query';
 import { TimeRange } from 'ui/timefilter/time_history';
 import { IndexPattern } from 'ui/index_patterns';
 import { IPrivate } from 'ui/private';
-import { StaticIndexPattern, Query } from 'src/legacy/core_plugins/data/public';
+import { StaticIndexPattern, Query, SavedQuery } from 'plugins/data';
+import { data } from 'plugins/data/setup';
```

Comment:
Hey @lizozom I'm trying to follow the pattern you established [here](https://github.com/elastic/kibana/pull/38894) but my editor is complaining that the module cannot be found. The file I'm working in seems to be the first Typescript file using this new path. Any idea why I'd get that error?